citadel.dme

```diff
@@ -739,6 +739,7 @@
 #include "code\game\area\asteroid_areas.dm"
 #include "code\game\area\Away Mission areas.dm"
 #include "code\game\area\Off Station Areas.dm"
+#include "code\game\area\Rift_Areas.dm"
```

Comment:
lowercase file names for DM please. (grandfather clause of existing files)

Reply:
Done and done

Reply:
Just remembered why I did this. For some fucking reason if I dont have this file capitalized in both the dme and in its name it fails linters
![image](https://user-images.githubusercontent.com/10539415/206594006-3a506820-0b07-4881-afce-0dadb17b95cd.png)

Reply:
I manually set it with github